src/main/java/duke/data/TaskDataBase.java

```diff
@@ -0,0 +1,87 @@
+package duke.data;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+import duke.tasks.Event;
+import duke.tasks.Deadline;
+import duke.exceptions.InvalidDateException;
+import duke.ui.Ui;
+
+/**
+ * The `TaskDataBase` class provides methods to load and save tasks to and from a file.
+ */
+public class TaskDataBase {
+
+    /**
+     * Loads tasks from a file and returns them as a list.
+     *
+     * @return A list of tasks loaded from the file.
+     * @throws IOException If an I/O error occurs while reading the file.
+     * @throws InvalidDateException If a date in the file is invalid or cannot be parsed.
+     */
+    public static List<Task> load() throws IOException, InvalidDateException {
```

Comment:
Yes this is the right way to do it, [stackoverflow](https://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java)